### DISCUSSION TITLE
Comparing the most recent values of a sequence , given  a subsequence value, againts,  the rest of the sequence

### DISCUSSION THREAD

**Discussion by dieguisgs** [TDAmeritrade/stumpy#912] [General]
Hi there,

Is tehre any option to generate the matrix profile giging two dataframes o arrays where the short one comprehends the latest values of the long array.

For the approach of konowing the most similar patterns of the last values of a Series given a subsequence arguement.

I see stumpy.stump is used for comparaing all the combiantions for an only dataframe.

Thanks,

Diego

**Comment by seanlaw** [comment]
@dieguisgs Thank you for your question and welcome to the STUMPY community. I may be misunderstanding your question so maybe you can share some code of what you are trying to accomplish. However, if I understand correctly, I think you may be looking for the [`stumpy.mass` function](https://stumpy.readthedocs.io/en/latest/api.html#stumpy.mass). See [this tutorial](https://stumpy.readthedocs.io/en/latest/Tutorial_Pattern_Matching.html) for more details and feel free to respond here with more questions.
